startup-script/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM debian:jessie
```

Comment:
Is jessie just for convenience? It seems like we shouldn't need a fully fledged OS underneath a simple startup script example. (also, smaller base image == smaller attack surface => less image rebuilding as vulnerabilities as found)